justfile

```diff
@@ -38,8 +38,13 @@ _image_profile := if profile == "debug" { "debug" } else { "release" }
 _dpdk_sys_container_repo := "ghcr.io/githedgehog/dpdk-sys"
 [private]
 _dpdk_sys_container_tag := dpdk_sys_commit
+
 [private]
 _libc_container := _dpdk_sys_container_repo + "/libc-env:" + _dpdk_sys_container_tag + "." + _image_profile
+
+[private]
+_doc_env_container := _dpdk_sys_container_repo + "/doc-env:" + _dpdk_sys_container_tag
+
```

Comment:
I think we removed the use of the `doc-env` container, this change is probably accidental?

Reply:
Yup, good catch Quentin. Will remove...